mcprep-build.py

```diff
@@ -0,0 +1,98 @@
+import os
```

Comment:
Ok now I'll admit I'm a little confused - should we be using this file for making builds? or just running something like `bpy-addon-build -b dev`? Is this just a mix of copied from bpy-addon-build..? Seems like this file is trying to perform the building itself

Reply:
I forgot to remove this file after switching to bpy-build